lib/ExpensiMark.js

```diff
@@ -261,7 +264,10 @@ export default class ExpensiMark {
                 name: 'quote',
                 regex: /\n?<(blockquote|q)(?:"[^"]*"|'[^']*'|[^'">])*>([\s\S]*?)<\/\1>(?![^<]*(<\/pre>|<\/code>))/gi,
                 replacement: (match, g1, g2) => {
-                    const resultString = g2.trim().split('\n').map(m => `> ${m}`).join('\n');
+                    const resultString = g2.replace(/(\[block\])+/g, '\n').trim()
```

Comment:
We also need the trim in a new line, sorry I missed it!

Reply:
updated